lib/pinyin/zhuyin_parsers.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Pinyin.ZhuYinParsers do
```

Comment:
I'd rename this to `ZhuYin.Parsers` and move the file to a (new) `zhuyin` directory. Also, if the capitalization of `ZhuYin` is correct, you should name this file `zhu_yin_parsers`, but I'm guessing it should be `Zhuyin`?

Reply:
> if the capitalization of ZhuYin is correct

Well that's debatable, whether you want to capitalize syllables or not. But I agree, I should do the same as for the Pinyin module and I moved it to a new directory.